src/client/features/enrichment/cy/cluster-label-categorization/flaggedwords.json

```diff
@@ -0,0 +1,9 @@
+[
```

Comment:
How did you come up with these? I feel like we could just run through the GMT file descriptions, rank the words (minus stop), look at the distribution and boot the top N. I'm sure other people have thought long about this.

Reply:
I found them in the bader lab  cloud word app repository